api/src/main/java/org/openmrs/validator/VisitValidator.java

```diff
@@ -47,6 +49,8 @@ public boolean supports(Class<?> clazz) {
 	 * @should fail if visit type is not set
 	 * @should fail if startDatetime is not set
 	 * @should fail if the endDatetime is before the startDatetime
+	 * @should fail if the startDatetime is before the birthDate
+	 * @should fail if the startDatetime is within an year of the patient's birthDate
```

Comment:
This line is definitely wrong. (I haven't looked at the code yet to see if it's only an incorrect comment.)